pageObjects/RegistrationPage.js

```diff
@@ -0,0 +1,54 @@
+import { test, expect } from "@playwright/test";
+
+export default class RegistrationPage {
```

Comment:
There is no registration page as well as login page

You can call it MainPage or WelcomePage or smth like that

Reply:
fixed